<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Grover Lab</title>
  <meta name="description" content="Grover Lab<br> Department of Bioengineering<br> University of California, Riverside<br> Riverside, CA 92521
">

  <link rel="stylesheet" href="/css/main.css">
  <link rel="canonical" href="http://groverlab.org/research/">
  <link rel="alternate" type="application/rss+xml" title="Grover Lab" href="http://groverlab.org/feed.xml">

  
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-34084184-1', 'auto');
  ga('send', 'pageview');

</script>


</head>


  <body>

    <header class="site-header">

  <div class="wrapper">

    <a class="site-title" href="/">Grover Lab</a> 


    <nav class="site-nav">
      <a href="#" class="menu-icon">
        <svg viewBox="0 0 18 15">
          <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
          <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
          <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
        </svg>
      </a>

      <div class="trigger">
          <a class="page-link" href="/">Home</a>
          <a class="page-link" href="/research">Research</a>
          <a class="page-link" href="/education">Education</a>
          <a class="page-link" href="/hnbfpr">HNBFPR</a>
          <a class="page-link" href="/people">People</a>
          <a class="page-link" href="/contact">Contact</a>
      </div>
    </nav>


  </div>

  <div class="wrapper">

  <p style="font-size:15px">Department of Bioengineering, University of California, Riverside</p>

  </div>
  

</header>


    <div class="page-content">
      <div class="wrapper">
        <div class="home">

  <h1>Research</h1>

  <ul class="post-list">
    
      <li>
      <br>
<!--         <span class="post-meta"></span>
 -->
<!--           <img src="/assets/" width="30%" align="left" style="margin:0px 20px 0px 0px"> -->

           
        <h2>
          <a class="post-link" href="/research/2016-06-21-mec.html">MECs: "Building Blocks" for Creating Biological and Chemical Instruments</a>
        </h2>
        <p>Douglas A. Hill, Lindsey E. Anderson, Casey J. Hill, Afshin Mostaghim, Victor G. J. Rodgers, and William H. Grover.  PLoS ONE 11(7): e0158706.  <a href="/assets/mec.pdf">PDF</a></p>

<p><img src="/assets/MEC_fig1.png" /></p>

<p>The development of new biological and chemical instruments for research and diagnostic applications is often slowed by the cost, specialization, and custom nature of these instruments.  New instruments are built from components that are drawn from a host of different disciplines and not designed to integrate together, and once built, an instrument typically performs a limited number of tasks and cannot be easily adapted for new applications. Consequently, the process of inventing new instruments is very inefficient, especially for researchers or clinicians in resource-limited settings.  To improve this situation, we propose that a family of <em>standardized multidisciplinary components</em> is needed, a set of “building blocks” that perform a wide array of different tasks and are designed to integrate together.  Using these components, scientists, engineers, and clinicians would be able to build custom instruments for their own unique needs quickly and easily.  In this work we present the foundation of this set of components, a system we call Multifluidic Evolutionary Components (MECs).  “Multifluidic” conveys the wide range of fluid volumes MECs operate upon (from nanoliters to milliliters and beyond); “multi” also reflects the multiple disciplines supported by the system (not only fluidics but also electronics, optics, and mechanics). “Evolutionary” refers to the design principles that enable the library of MEC parts to easily grow and adapt to new applications.  Each MEC “building block” performs a fundamental function that is commonly found in biological or chemical instruments, functions like valving, pumping, mixing, controlling, and sensing.  Each MEC also has a unique symbol linked to a physical definition, which enables instruments to be designed rapidly and efficiently using schematics.  As a proof-of-concept, we use MECs to build a variety of instruments, including a fluidic routing and mixing system capable of manipulating fluid volumes over five orders of magnitude, an acid-base titration instrument suitable for use in schools, and a bioreactor suitable for maintaining and analyzing cell cultures in research and diagnostic applications.  These are the first of many instruments that can be built by researchers, clinicians, and students using the MEC system.</p>

      </li>
    
      <li>
      <br>
<!--         <span class="post-meta"></span>
 -->
<!--           <img src="/assets/" width="30%" align="left" style="margin:0px 20px 0px 0px"> -->

           
        <h2>
          <a class="post-link" href="/research/2016-01-01-orientation-based-microfluidics.html">Orientation-based control of microfluidics</a>
        </h2>
        <p>Nazila Norouzi, Heran C. Bhakta, and William H. Grover.  PLoS ONE 11(3): e0149259  <a href="/assets/orientation-based-microfluidics.pdf">PDF</a></p>

<p><img src="/assets/orientation-based-microfluidics.png" /></p>

<p>Most microfluidic chips utilize off-chip hardware (syringe pumps, computer-controlled solenoid valves, pressure regulators, etc.) to control fluid flow on-chip. This expensive, bulky, and power-consuming hardware severely limits the utility of microfluidic instruments in resource-limited or point-of-care contexts, where the cost, size, and power consumption of the instrument must be limited. In this work, we present a technique for on-chip fluid control that requires no off-chip hardware. We accomplish this by using inert compounds to change the density of one fluid in the chip. If one fluid is made 2% more dense than a second fluid, when the fluids flow together under laminar flow the interface between the fluids quickly reorients to be orthogonal to Earth’s gravitational force. If the channel containing the fluids then splits into two channels, the amount of each fluid flowing into each channel is precisely determined by the angle of the channels relative to gravity. Thus, any fluid can be routed in any direction and mixed in any desired ratio on-chip simply by holding the chip at a certain angle. This approach allows for sophisticated control of on-chip fluids with no off-chip control hardware, significantly reducing the cost of microfluidic instruments in point-of-care or resource-limited settings.</p>


      </li>
    
      <li>
      <br>
<!--         <span class="post-meta"></span>
 -->
<!--           <img src="/assets/" width="30%" align="left" style="margin:0px 20px 0px 0px"> -->

           
        <h2>
          <a class="post-link" href="/research/2015-10-08-flow-layer-design.html">Flow-layer physical design for microchips based on monolithic membrane valves</a>
        </h2>
        <p>Jeffrey McDaniel, Brian Crites, Philip Brisk, and William H. Grover.  <em>IEEE Design and Test</em> 32 (6), 51-59 (2015).  <a href="/assets/flow-layer-design.pdf">PDF</a></p>

<p><img src="/assets/flow-layer-design.png" /></p>

<p>This article introduces a software toolchain for physical design and layout for the flow layer of microfluidic LoCs based on integrated microvalve technology. A case study shows that it can automatically produce layouts for the Mars Organic Analyzer LoC to detect biomolecules in soil on Mars.</p>


      </li>
    
      <li>
      <br>
<!--         <span class="post-meta"></span>
 -->
<!--           <img src="/assets/" width="30%" align="left" style="margin:0px 20px 0px 0px"> -->

           
        <h2>
          <a class="post-link" href="/research/2015-01-01-3d-tox.html">Assessing and Reducing the Toxicity of 3D-Printed Parts</a>
        </h2>
        <p>Shirin Mesbah Oskui, Graciel Diamante, Chunyang Liao, Wei Shi, Jay Gan, Daniel Schlenk, and William H. Grover, Environmental Science and Technology Letters, Article ASAP.  <a href="/assets/3d-tox.pdf">PDF</a></p>

<p><img src="/assets/3d-tox.png" /></p>

<p>3D printing is gaining popularity by providing a tool for fast, cost-effective, and highly customizable fabrication. However, little is known about the toxicity of 3D-printed objects. In this work, we assess the toxicity of printed parts from two main classes of commercial 3D printers, fused deposition modeling and stereolithography. We assessed the toxicity of these 3D-printed parts using zebrafish (<em>Danio rerio</em>), a widely used model organism in aquatic toxicology. Zebrafish embryos were exposed to 3D-printed parts and monitored for rates of survival, hatching, and developmental abnormalities. We found that parts from both types of printers were measurably toxic to zebrafish embryos, with STL-printed parts significantly more toxic than FDM-printed parts. We also developed a simple post-printing treatment (exposure to ultraviolet light) that largely mitigates the toxicity of the STL-printed parts. Our results call attention to the need for strategies for the safe disposal of 3D-printed parts and printer waste materials.</p>


      </li>
    
      <li>
      <br>
<!--         <span class="post-meta"></span>
 -->
<!--           <img src="/assets/" width="30%" align="left" style="margin:0px 20px 0px 0px"> -->

           
        <h2>
          <a class="post-link" href="/research/2011-01-01-cell-density.html">Measuring single-cell density</a>
        </h2>
        <p>William H. Grover, Andrea K. Bryan, Monica Diez-Silva, Subra Suresh, John M. Higgins, and Scott R. Manalis, <em>Proceedings of the National Academy of Sciences of the United States of America</em> 108 (27), 10992-10996 (2011).  <a href="/assets/cell-density.pdf">PDF</a></p>

<p><img src="/assets/cell-density.png" /></p>

<p>We have used a microfluidic mass sensor to measure the density of single living cells. By weighing each cell in two fluids of different densities, our technique measures the single-cell mass, volume, and density of approximately 500 cells per hour with a density precision of 0.001 g/mL. We observe that the intrinsic cell-to-cell variation in density is nearly 100-fold smaller than the mass or volume variation.  As a result, we can measure changes in cell density indicative of cellular processes that would be otherwise undetectable by mass or volume measurements. Here we demonstrate this with four examples: identifying <em>P. falciparum</em> malaria-infected erythrocytes in a culture, distinguishing transfused blood cells from a patient’s own blood, identifying irreversibly-sickled cells in a sickle cell patient, and identifying leukemia cells in the early stages of responding to a drug treatment.  These demonstrations suggest that the ability to measure single cell density will provide valuable insights into cell state for a wide range of biological processes.</p>


      </li>
    
      <li>
      <br>
<!--         <span class="post-meta"></span>
 -->
<!--           <img src="/assets/cell-growth.png" width="30%" align="left" style="margin:0px 20px 0px 0px"> -->

           
        <h2>
          <a class="post-link" href="/research/2010-01-01-cell-growth.html">Using buoyant mass to measure the growth of single cells</a>
        </h2>
        <p>Michel Godin, Francisco Feijo Delgado, Sungmin Son, William H. Grover, Andrea K. Bryan, Amit Tzur, Paul Jorgensen, Kris Payer, Alan D. Grossman, Marc W. Kirschner, and Scott R. Manalis, <em>Nature Methods</em> 7, 387-390 (2010).  <a href="/assets/cell-growth.pdf">PDF</a></p>

<p><img src="/assets/cell-growth.png" /></p>

<p>We used a suspended microchannel resonator (SMR) combined with picoliter-scale microfluidic control to measure buoyant mass and determine the “instantaneous” growth rates of individual cells. The SMR measures mass with femtogram precision, allowing rapid determination of the growth rate in a fraction of a complete cell cycle. We found that for individual cells of Bacillus subtilis, <em>Escherichia coli</em>, <em>Saccharomyces cerevisiae</em> and mouse lymphoblasts, heavier cells grew faster than lighter cells.</p>


      </li>
    
      <li>
      <br>
<!--         <span class="post-meta"></span>
 -->
<!--           <img src="/assets/" width="30%" align="left" style="margin:0px 20px 0px 0px"> -->

           
        <h2>
          <a class="post-link" href="/research/2008-01-01-teflon-valves.html">Teflon films for chemically-inert microfluidic valves and pumps</a>
        </h2>
        <p>William H. Grover, Marcio G. von Muhlen, and Scott R. Manalis, Lab on a Chip 8 (6), 913-918 (2008).  <a href="/assets/teflon-valves.pdf">PDF</a></p>

<p><img src="/assets/teflon-valves.png" /></p>

<p>We present a simple method for fabricating chemically-inert Teflon microfluidic valves and pumps in glass microfluidic devices. These structures are modeled after monolithic membrane valves and pumps that utilize a featureless polydimethylsiloxane (PDMS) membrane bonded between two etched glass wafers. The limited chemical compatibility of PDMS has necessitated research into alternative materials for microfluidic devices. Previous work has shown that spin-coated amorphous fluoropolymers and Teflon-fluoropolymer laminates can be fabricated and substituted for PDMS in monolithic membrane valves and pumps for space flight applications. However, the complex process for fabricating these spin-coated Teflon films and laminates may preclude their use in many research and manufacturing contexts. As an alternative, we show that commercially-available fluorinated ethylene-propylene (FEP) Teflon films can be used to fabricate chemically-inert monolithic membrane valves and pumps in glass microfluidic devices. The FEP Teflon valves and pumps presented here are simple to fabricate, function similarly to their PDMS counterparts, maintain their performance over extended use, and are resistant to virtually all chemicals. These structures should facilitate lab-on-a-chip research involving a vast array of chemistries that are incompatible with native PDMS microfluidic devices.</p>

      </li>
    
      <li>
      <br>
<!--         <span class="post-meta"></span>
 -->
<!--           <img src="/assets/" width="30%" align="left" style="margin:0px 20px 0px 0px"> -->

           
        <h2>
          <a class="post-link" href="/research/2007-01-01-pneumatic-logic.html">Micropneumatic digital logic structures for integrated microdevice computation and control</a>
        </h2>
        <p>Erik C. Jensen, William H. Grover, and Richard A. Mathies, Journal of Microelectromechanical Systems 16 (6), 1378-1385 (2007).  <a href="/assets/pneumatic-logic.pdf">PDF</a></p>

<p><img src="/assets/pneumatic-logic.png" /></p>

<p>It is shown that microfabricated polydimethylsiloxane membrane valve structures can be configured to function as transistors in pneumatic digital logic circuits. Using the analogy with metal-oxide-semiconductor field-effect transistor circuits, networks of pneumatically actuated microvalves are designed to produce pneumatic digital logic gates (AND, OR, NOT, NAND, and XOR). These logic gates are combined to form 4- and 8-bit ripple-carry adders as a demonstration of their universal pneumatic computing capabilities. Signal propagation through these pneumatic circuits is characterized, and an amplifier circuit is demonstrated for improved signal transduction. Propagation of pneumatic carry information through the 8-bit adder is complete within 1.1 s, demonstrating the feasibility of integrated temporal control of pneumatic actuation systems. Integrated pneumatic logical systems reduce the number of off-chip controllers required for lab-on-a-chip and microelectromechanical system devices, allowing greater complexity and portability. This technology also enables the development of digital pneumatic computing and logic systems that are immune to electromagnetic interference.</p>

      </li>
    
      <li>
      <br>
<!--         <span class="post-meta"></span>
 -->
<!--           <img src="/assets/" width="30%" align="left" style="margin:0px 20px 0px 0px"> -->

           
        <h2>
          <a class="post-link" href="/research/2006-01-01-serial-dilution.html">Microfluidic serial dilution circuit</a>
        </h2>
        <p>Brian M. Paegel, William H. Grover, Alison M. Skelley, Richard A. Mathies, Gerald F. Joyce, Analytical Chemistry 78 (21), 7522-7527 (2006).  <a href="/assets/serial-dilution.pdf">PDF</a></p>

<p><img src="/assets/serial-dilution.png" /></p>

<p><em>In vitro</em> evolution of RNA molecules requires a method for executing many consecutive serial dilutions. To solve this problem, a microfluidic circuit has been fabricated in a three-layer glass-PDMS-glass device. The 400-nL serial dilution circuit contains five integrated membrane valves: three two-way valves arranged in a loop to drive cyclic mixing of the diluent and carryover, and two bus valves to control fluidic access to the circuit through input and output channels. By varying the valve placement in the circuit, carryover fractions from 0.04 to 0.2 were obtained. Each dilution process, which is composed of a diluent flush cycle followed by a mixing cycle, is carried out with no pipetting, and a sample volume of 400 nL is sufficient for conducting an arbitrary number of serial dilutions. Mixing is precisely controlled by changing the cyclic pumping rate, with a minimum mixing time of 22 s. This microfluidic circuit is generally applicable for integrating automated serial dilution and sample preparation in almost any microfluidic architecture.</p>

      </li>
    
      <li>
      <br>
<!--         <span class="post-meta"></span>
 -->
<!--           <img src="/assets/" width="30%" align="left" style="margin:0px 20px 0px 0px"> -->

           
        <h2>
          <a class="post-link" href="/research/2006-01-01-latching-valves.html">Development and multiplexed control of latching pneumatic valves using microfluidic logical structures</a>
        </h2>
        <p>William H. Grover, Robin H.C. Ivester, Erik C. Jensen, and Richard A. Mathies, Lab on a Chip 6 (5), 623-631 (2006). <a href="/assets/latching-valves.pdf">PDF</a></p>

<p><img src="/assets/latching-valves.png" /></p>

<p>Novel latching microfluidic valve structures are developed, characterized, and controlled independently using an on-chip pneumatic demultiplexer. These structures are based on pneumatic monolithic membrane valves and depend upon their normally-closed nature. Latching valves consisting of both three- and four-valve circuits are demonstrated. Vacuum or pressure pulses as short as 120 ms are adequate to hold these latching valves open or closed for several minutes. In addition, an on-chip demultiplexer is demonstrated that requires only n pneumatic inputs to control 2(n-1) independent latching valves. These structures can reduce the size, power consumption, and cost of microfluidic analysis devices by decreasing the number of off-chip controllers. Since these valve assemblies can form the standard logic gates familiar in electronic circuit design, they should be useful in developing complex pneumatic circuits.</p>

      </li>
    
      <li>
      <br>
<!--         <span class="post-meta"></span>
 -->
<!--           <img src="/assets/" width="30%" align="left" style="margin:0px 20px 0px 0px"> -->

           
        <h2>
          <a class="post-link" href="/research/2005-01-01-microfluidic-processor.html">An integrated microfluidic processor for single nucleotide polymorphism-based DNA computing</a>
        </h2>
        <p>William H. Grover and Richard A. Mathies, Lab on a Chip 5 (10), 1033-1040 (2005).  <a href="/assets/microfluidic-processor.pdf">PDF</a></p>

<p><img src="/assets/microfluidic-processor.png" /></p>

<p>An integrated microfluidic processor is developed that performs molecular computations using single nucleotide polymorphisms (SNPs) as binary bits. A complete population of fluorescein-labeled DNA “answers” is synthesized containing three distinct polymorphic bases; the identity of each base (A or T) is used to encode the value of a binary bit (TRUE or FALSE). Computation and readout occur by hybridization to complementary capture DNA oligonucleotides bound to magnetic beads in the microfluidic device. Beads are loaded into sixteen capture chambers in the processor and suspended in place by an external magnetic field. Integrated microfluidic valves and pumps circulate the input DNA population through the bead suspensions. In this example, a program consisting of a series of capture/rinse/release steps is executed and the DNA molecules remaining at the end of the computation provide the solution to a three-variable, four-clause Boolean satisfiability problem. The improved capture kinetics, transfer efficiency, and single-base specificity enabled by microfluidics make our processor well-suited for performing larger-scale DNA computations.</p>


      </li>
    
      <li>
      <br>
<!--         <span class="post-meta"></span>
 -->
<!--           <img src="/assets/" width="30%" align="left" style="margin:0px 20px 0px 0px"> -->

           
        <h2>
          <a class="post-link" href="/research/2005-01-01-mars-biomarkers.html">Development and evaluation of a microdevice for amino acid biomarker detection and analysis on Mars</a>
        </h2>
        <p>Alison M. Skelley, James R. Scherer, Andrew D. Aubrey, William H. Grover, Robin H.C. Ivester, Pascale Ehrenfreund, Frank J. Grunthaner, Jeffrey L. Bada, Richard A. Mathies, Proceedings of the National Academy of Sciences of the United States of America 102 (4), 1041-1046 (2005).   <a href="/assets/mars-biomarkers.pdf">PDF</a></p>

<p><img src="/assets/mars-biomarkers.png" /></p>

<p>The Mars Organic Analyzer (MOA), a microfabricated capillary electrophoresis (CE) instrument for sensitive amino acid biomarker analysis, has been developed and evaluated. The microdevice consists of a four-wafer sandwich combining glass CE separation channels, microfabricated pneumatic membrane valves and pumps, and a nanoliter fluidic network. The portable MOA instrument integrates high voltage CE power supplies, pneumatic controls, and fluorescence detection optics necessary for field operation. The amino acid concentration sensitivities range from micromolar to 0.1 nM, corresponding to part-per-trillion sensitivity. The MOA was first used in the lab to analyze soil extracts from the Atacama Desert, Chile, detecting amino acids ranging from 10-600 parts per billion. Field tests of the MOA in the Panoche Valley, CA, successfully detected amino acids at 70 parts per trillion to 100 parts per billion in jarosite, a sulfate-rich mineral associated with liquid water that was recently detected on Mars. These results demonstrate the feasibility of using the MOA to perform sensitive in situ amino acid biomarker analysis on soil samples representative of a Mars-like environment.</p>


      </li>
    
      <li>
      <br>
<!--         <span class="post-meta"></span>
 -->
<!--           <img src="/assets/" width="30%" align="left" style="margin:0px 20px 0px 0px"> -->

           
        <h2>
          <a class="post-link" href="/research/2003-01-01-monolithic-membrane-valves.html">Monolithic membrane valves and diaphragm pumps for practical large-scale integration into glass microfluidic devices</a>
        </h2>
        <p>William H. Grover, Alison M. Skelley, Chung N. Liu, Eric T. Lagally, Richard A. Mathies, Sensors and Actuators B 89 (3), 315-323 (2003).  <a href="/assets/monolithic-membrane-valves.pdf">PDF</a></p>

<p><img src="/assets/monolithic-membrane-valves.png" /></p>

<p>Monolithic elastomer membrane valves and diaphragm pumps suitable for large-scale integration into glass microfluidic analysis devices are fabricated and characterized. Valves and pumps are fabricated by sandwiching an elastomer membrane between etched glass fluidic channel and manifold wafers. A three-layer valve and pump design features simple non-thermal device bonding and a hybrid glass-PDMS fluidic channel; a four-layer structure includes a glass fluidic system with minimal fluid-elastomer contact for improved chemical and biochemical compatibility. The pneumatically actuated valves have less than 10 nl dead volumes, can be fabricated in dense arrays, and can be addressed in parallel via an integrated manifold. The membrane valves provide flow rates up to 380 nL/s at 30 kPa driving pressure and seal reliably against fluid pressures as high as 75 kPa. The diaphragm pumps are self-priming, pump from a few nanoliters to a few microliters per cycle at overall rates from 1 to over 100 nl/s, and can reliably pump against 42 kPa pressure heads. These valves and pumps provide a facile and reliable integrated technology for fluid manipulation in complex glass microfluidic and electrophoretic analysis devices.</p>


      </li>
    
  </ul>

  <p class="rss-subscribe">subscribe <a href="/feed.xml">via RSS</a></p>

</div>

      </div>
    </div>

    <footer class="site-footer">

  <div class="wrapper">

    <h2 class="footer-heading">Grover Lab</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li>Grover Lab</li>
          <li><a href="mailto:wgrover@engr.ucr.edu">wgrover@engr.ucr.edu</a></li>
        </ul>
      </div>

      <div class="footer-col footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/groverlab"><span class="icon icon--github"><svg viewBox="0 0 16 16"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">groverlab</span></a>

          </li>
          

          
          <li>
            <a href="https://twitter.com/wgrover"><span class="icon icon--twitter"><svg viewBox="0 0 16 16"><path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/></svg>
</span><span class="username">wgrover</span></a>

          </li>
          
        </ul>
      </div>

      <div class="footer-col footer-col-3">
        <p>Grover Lab<br> Department of Bioengineering<br> University of California, Riverside<br> Riverside, CA 92521
</p>
      </div>
    </div>

  </div>

</footer>


  </body>

</html>
